javascript/builtins/Array.json

```diff
@@ -105,6 +105,58 @@
             }
           }
         },
+        "at": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/at",
+            "spec_url": "https://tc39.es/proposal-relative-indexing-method/",
+            "support": {
+              "chrome": {
+                "version_added": "89"
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "85"
```

Comment:
I think per convention this needs to be false, because it's only enabled in Firefox Nightly.

Reply:
I'm going to delve a little deeper as the chrome version added is canary, so the same rules should apply one assumes... I'll read the guidelines ☺️

Reply:
Firefox does not ship to release, so yes, it should be false there.

Chrome versions are fine as https://www.chromestatus.com/feature/6123640410079232 says it is enabled by default and thus will ride these train to release.

Reply:
I am happy to make these changes @Elchi3 as I am unsure as to what we add to BCD as far as future versions go... however at() is also enabled by default in FF85 - does it not follow the same rules?

Reply:
> however at() is also enabled by default in FF85 - does it not follow the same rules?

So, https://bugzilla.mozilla.org/show_bug.cgi?id=1681371 is not yet resolved fixed, and so I think it is not yet enabled by default.

> I am unsure as to what we add to BCD as far as future versions go...

Once https://bugzilla.mozilla.org/show_bug.cgi?id=1681371 is implemented in Firefox we will have a version number we can use.

Reply:
Thank you @Elchi3 🙏 the more I know...